src/components/Header/Header.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Header = () => (
+  <section>
+    <h1>Meet the Queens</h1>
+  </section>
```

Comment:
Looks like there's no way to go back to the main list in this header. You could probably add a simple navigation bar with an a tag to get it done:
```
 <section>
    <h1>Meet the Queens</h1>
    <nav>
      <a href='/'>Home</a>
    </nav>
  </section>
```